helm/kubernetes-cluster-autoscaler-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    giantswarm.io/service-type: "{{ .Values.serviceType }}"
+    k8s-app: {{ .Values.name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: {{ .Values.name }}
+  template:
+    metadata:
+      labels:
+        giantswarm.io/service-type: "{{ .Values.serviceType }}"
+        k8s-app: {{ .Values.name }}
+    spec:
+      serviceAccountName: {{ .Values.name }}
+      tolerations:
+      - effect: NoSchedule
+        operator: "Equal"
+        value: "true"
+        key: node-role.kubernetes.io/master
+      nodeSelector:
+        kubernetes.io/role: master
+      containers:
+        - image: k8s.gcr.io/cluster-autoscaler:v1.12.1
+          name: {{ .Values.name }}
+          resources:
+            limits:
+              cpu: 100m
+              memory: 300Mi
+            requests:
+              cpu: 100m
+              memory: 300Mi
+          command:
+            - ./cluster-autoscaler
+            - --v=4
+            - --stderrthreshold=info
+            - --cloud-provider=aws
+            - --skip-nodes-with-local-storage=false
+            - --nodes={{ .Values.nodes.min }}:{{ .Values.nodes.max }}:{{ .Values.nodes.asg.name }}
```

Comment:
Ok, so how do we provide multiple ASGs?

Reply:
I believe it would just be appended like `1:10:name1:1:20:name2` - we only use one ASG right now AFAIK. So this should be fine for now and we can improve this later.

Reply:
Only one ASG now and we need to figure out multi-ASG later with node pool story.